### PR TITLE
Multiple bindings - Fix README formatting regression

### PR DESCRIPTION
**Pull request by mlobstein**
Fix the formatting in the README for multiple bindings where two consecutive spaces were used to create a line break. Regression of #17422.

### CODE REVIEW

**Comment by jlaur** [comment]
For reference: https://www.markdownguide.org/basic-syntax/#line-break-best-practices

I was not aware of that. [Before](https://www.openhab.org/addons/bindings/panasonicbdp/) #17422:

image

[After](https://next.openhab.org/addons/bindings/panasonicbdp/):

image

@mlobstein - thanks for the fix. Do you actually want space between the paragraphs, or just a line break? If you want just a line break (as before), I guess you would need to either add the two spaces again, or `<br>`?

**Comment by mlobstein** [comment]
> 
> @mlobstein - thanks for the fix. Do you actually want space between the paragraphs, or just a line break? If you want just a line break (as before), I guess you would need to either add the two spaces again, or `<br>`?

I fixed the MR description since it was not clear. I am fine with leaving this as is and not using two consecutive spaces to make a line break since they will likely get flagged again in future cleanup efforts.

**Comment by jlaur** [comment]
> I am fine with leaving this as is and not using two consecutive spaces to make a line break since they will likely get flagged again in future cleanup efforts.

Thanks for the feedback. One more thing: I see you are binding maintainer for the five README's fixed in this PR. Just to be sure - was it only those five README's having this regression out of the 209 changed, or did you only check those for which you are code owner?

File: bundles/org.openhab.binding.kaleidescape/README.md [hunk 1]
```diff
@@ -13,10 +13,10 @@ All movie player components including the original K-Player series, M Class Play
 It is important to choose the correct thing type to ensure the available channels are correct for the component being used.
 
 The supported thing types are:
-`player` Any KPlayer, M Class [M300, M500, M700] or Cinema One 1st Gen player
-`cinemaone` Cinema One (2nd Gen)
-`alto`
-`strato` Includes Strato, Strato S, Strato C or Strato V
+- `player` Any KPlayer, M Class [M300, M500, M700] or Cinema One 1st Gen player
```
Comment (lsiepel) [inline bundles/org.openhab.binding.kaleidescape/README.md:16]: If this is changed to a list, there should be an empty line:
MD032/blanks-around-lists: Lists should be surrounded by blank linesmarkdownlint[MD032](https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md032.md)

```suggestion

- `player` Any KPlayer, M Class [M300, M500, M700] or Cinema One 1st Gen player
```

File: bundles/org.openhab.binding.panasonicbdp/README.md [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
 If the connection to the player originates from a different subnet, 404 response errors are sent in response to all requests.
 
 To enable network remote control of the Blu-ray model players, configure the following settings:
-**Player Settings/Network/Network Settings/Remote Device Settings**
+- **Player Settings/Network/Network Settings/Remote Device Settings**
+
 Then make sure you have the following values set:
-**Remote Device Operation: On**
-**Registration Type: Automatic**
+- **Remote Device Operation: On**
```
Comment (lsiepel) [inline bundles/org.openhab.binding.panasonicbdp/README.md:16]: See previous list comment.
```suggestion

- **Remote Device Operation: On**
```

File: bundles/org.openhab.binding.panasonicbdp/README.md [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
+- **Registration Type: Automatic**
 
 For the UHD models, Voice Control must be enabled for the player's http interface to be active:
-**Player Settings/Network/Voice Control: On**
+- **Player Settings/Network/Voice Control: On**
```
Comment (lsiepel) [inline bundles/org.openhab.binding.panasonicbdp/README.md:20]: See previous list comment.
```suggestion

- **Player Settings/Network/Voice Control: On**
```

File: bundles/org.openhab.binding.panasonicbdp/README.md [hunk 4]
```diff
@@ -1,27 +1,32 @@
...
 
 To enable the binding to control the player while off (network active while off), Quick Start mode must be On as follows:
-**Player Settings/System/Quick Start: On**
+- **Player Settings/System/Quick Start: On**
```
Comment (lsiepel) [inline bundles/org.openhab.binding.panasonicbdp/README.md:23]: See previous list comment.
```suggestion

- **Player Settings/System/Quick Start: On**
```

File: bundles/org.openhab.binding.roku/README.md [hunk 5]
```diff
@@ -57,37 +57,37 @@ Some Notes:
...
-Up
-Down
-Back
-InstantReplay
-Info
-Backspace
-Search
-Enter
-FindRemote
+- Home
```
Comment (lsiepel) [inline bundles/org.openhab.binding.roku/README.md:60]: See previous list comment.
```suggestion

- Home
```

File: bundles/org.openhab.binding.roku/README.md [hunk 6]
```diff
@@ -57,37 +57,37 @@ Some Notes:
...
-VolumeMute
-InputTuner
-InputHDMI1
-InputHDMI2
-InputHDMI3
-InputHDMI4
-InputAV1
-PowerOff
-POWERON _(NOTE: POWERON needs to be completely capitalized due to a bug with older Roku devices)_
+- ChannelUp
```
Comment (lsiepel) [inline bundles/org.openhab.binding.roku/README.md:78]: See previous list comment.
```suggestion

- ChannelUp
```

File: bundles/org.openhab.binding.tivo/README.md [hunk 7]
```diff
@@ -66,61 +66,61 @@ All devices support the following channels:
...
-ASPECT_CORRECTION_WIDE_ZOOM
-VIDEO_MODE_FIXED_480i
-VIDEO_MODE_FIXED_480p
-VIDEO_MODE_FIXED_720p
-VIDEO_MODE_FIXED_1080i
-VIDEO_MODE_HYBRID
-VIDEO_MODE_HYBRID_720p
-VIDEO_MODE_HYBRID_1080i
-VIDEO_MODE_NATIVE
+- UP
```
Comment (lsiepel) [inline bundles/org.openhab.binding.tivo/README.md:69]:
```suggestion

- UP
```

File: bundles/org.openhab.binding.vizio/README.md [hunk 8]
```diff
@@ -71,37 +71,37 @@ The following channels are available:
...
-Info
-Menu
-Home
-Exit
-Smartcast
-ccToggle
-PictureMode
-WideMode
-WideToggle
+- PowerOn
```
Comment (lsiepel) [inline bundles/org.openhab.binding.vizio/README.md:74]:
```suggestion

- PowerOn
```

**Review by lsiepel — CHANGES_REQUESTED** [review]
Thanks for correcting the previous cleanup. This sometimes happens to bulk PR's, sorry.

Have one comment and one question. The list need a empty line before and after the list. Also check if you want the maintain the double star emphasis for the lists, not sure if they are still usefull in all cases.

**Comment by mlobstein** [comment]
> > I am fine with leaving this as is and not using two consecutive spaces to make a line break since they will likely get flagged again in future cleanup efforts.
> 
> Thanks for the feedback. One more thing: I see you are binding maintainer for the five README's fixed in this PR. Just to be sure - was it only those five README's having this regression out of the 209 changed, or did you only check those for which you are code owner?

I only checked the ones where I am the owner since I knew what they had looked like before.

**Comment by jlaur** [comment]
> I only checked the ones where I am the owner since I knew what they had looked like before.

@lsiepel - you might then want to somehow check for line endings with two spaces in other changed files in the original PR.

**Comment by lsiepel** [comment]
> > I only checked the ones where I am the owner since I knew what they had looked like before.
> 
> @lsiepel - you might then want to somehow check for line endings with two spaces in other changed files in the original PR.

Yes, it’s in my list now.

**Review by lsiepel — APPROVED** [review]
Thanks for correcting. LGTM

**Pull request merged**